src/runtime_src/core/tools/common/ReportCu.cpp

```diff
@@ -203,6 +205,9 @@ populate_cus_new(const xrt_core::device *device)
     ptCu.put( "base_address",   "0x0");
     ptCu.put( "usage",          stat.usages);
     ptCu.put( "type", enum_to_str(cu_type::PS));
+    ptCu.put( "succ_return",    stat.succ_cnt);

```

Comment:
better to use full names like: successful_return etc

Reply:
Done